src/BKTClient.ts

```diff
@@ -310,7 +310,10 @@ export class BKTClientImpl implements BKTClient {
           ApiId.GET_EVALUATIONS,
           component.config().featureTag,
           result.error,
-        )
+      ).catch((err) => {
+        /* ignore error from the storage layer */
+        console.debug('BKTClient: Storage layer error in fetchEvaluations::trackFailure', err)
```

Comment:
We should print an error. Also, if we have the DebugLogger, we should use it instead.